frontend/src/components/TextEditor.jsx

```diff
@@ -6,13 +6,17 @@ import { useEffect, useRef } from "react";
 import { withHistory } from "slate-history";
 import { useSetRecoilState } from "recoil";
 
-import Toolbar from "@/components/common/ToolBar/Toolbar";
+import Toolbar from "@/components/common/Toolbar/Toolbar";
```

Comment:
"@/components/common/Toolbar/Toolbar" change the first `Toolbar` to `ToolBar` (capital B), otherwise it breaks on case-sensitive systems so our deploy breaks lol